cypress/e2e/CMS/$CancerTypeHomePage/index.js

```diff
@@ -163,5 +140,121 @@ And('user selects {string} from draft {string} dropdown', (dropdown, option) =>
+    cy.get('div[id*="pdq-links-current-items"]').then($el => {
+        summaryTitle = $el[0].innerText
+    })
+})
+
+Then('the PDQ link is matching the earlier selected PDQ link', () => {
+    cy.get('div[class*="cthp-treatment"] a').then($el => {
+        const title = $el[0].innerText;
+        expect(summaryTitle).to.include(title);
+    })
+})
 
+let summaryVideo;
+And('user remembers the title of selected video for further verification', () => {
+    cy.get("div[id*='cthp-video-current-']").then($el => {
+        summaryVideo = $el[0].innerText
+    })
+})
+
+Then('the video is matching the earlier selected video', () => {
+    cy.get('div[class*="cthp-screening"] img').then($el => {
+        const video = $el[0].innerText;
+        expect(summaryVideo).to.include(video);
+    })
+})
+
+let summaryPageTitle;
+And('user remembers title of selected Cancer Research List Page for future verification', () => {
+    cy.get("div[class*='cthp-research'] div[class*='label']").then($el => {
+        summaryPageTitle = $el[0].innerText
+    })
+})
+
+Then('the Cancer Research List page title is matching the earlier selected Cancer Research List page title', () => {
+    cy.get("h1").should('include.text', summaryPageTitle)
+
+})
+
+And('user clicks on {string} dropdown', (dropdown) => {
+    cy.get(`div[class='more-info list cthp-dropdown'] label:contains("${dropdown}")`).click({ force: true })
+})
+
+And('the following more info links are displayed', (dataTable) => {
+    for (const { title, url } of dataTable.hashes()) {
+        const replacedTestSiteSection = url.replace("{TEST_SITE_SECTION}", siteSection);
+        cy.get(`ul a[class='title']:contains("${title}")`).should('be.visible').and('have.attr', 'href', replacedTestSiteSection)
+    }
+})
+
+And('cthp causes card has a link {string} with href {string}', (linkText, href) => {
+    const replacedTestSiteSection = href.replace("{TEST_SITE_SECTION}", siteSection);
+    cy.get("div[class='equalheight bgWhite cthp-causes'] a").should('have.text', linkText)
+    cy.get("div[class='equalheight bgWhite cthp-causes'] a").should('have.attr', 'href', replacedTestSiteSection)
+})
+
+And('cthp survival card has a link {string} with href {string}', (title, href) => {
+    cy.get(`div[class='equalheight bgWhite cthp-survival'] a:contains("${title}")`).should('be.visible').and('have.attr', 'href', href)
+})
+
+And('cthp screening card has a link {string} with href {string}', (linkText, href) => {
+    const replacedTestSiteSection = href.replace("{TEST_SITE_SECTION}", siteSection);
+    cy.get("div[class='equalheight bgWhite cthp-screening'] a").should('have.text', linkText)
+    cy.get("div[class='equalheight bgWhite cthp-screening'] a").should('have.attr', 'href', replacedTestSiteSection)
+})
+
+And('cthp general card has description that is not empty', () => {
+    cy.get("div[class='equalheight bgWhite cthp-general'] p").should('be.visible')
```

Comment:
update locator
also, just checking that it is visible is not enough - you should add that it is not empty